app/resources/documentation/manual/index.md

```diff
@@ -145,9 +145,9 @@ There are two types of viewports: 3D viewports and 2D viewports. TrenchBroom giv
 
 No. of Viewports    Cycling View         Cycling Order
 ----------------    ------------         -------------
-1                   Single view          3D > XY > XZ > YZ
-2                   Right view           XY > XZ > YZ
-3                   Bottom right view    XZ > YZ
+1                   Single view          3D XY XZ YZ
```

Comment:
These have meaning, you can't just delete them. I'm assuming this was an accident?

Reply:
Likewise for a bunch of other changes in this commit.

Reply:
You're right. I did a massive search / replace deleting `>` character by mistake. I'll revert that change.